tests/jiradeps/test_main.py

```diff
@@ -38,14 +46,16 @@ def test_cli(mocker, config, epics, stories):
     mocker.patch('jiradeps.main.get_jira_session', return_value=MagicMock(
         search_issues=MagicMock(side_effect=[epics, stories])))
 
-    runner = CliRunner()
-    with runner.isolated_filesystem():
-        result = runner.invoke(main.cli,
-                               ['--password', 'test', '-g', 'ABC-123'],
-                               catch_exceptions=False)
-        # the output file can be gathered from the log if needed:
-        # print('\n' + result.output.split()[-1])
+    result = click_runner.invoke(main.cli,
+                                 ['-f', 'test', '-p', 'test', '-g', 'ABC-123'],
+                                 catch_exceptions=False)
     assert result.exit_code == 0
     assert 'loaded 1 epics' in result.output
     assert 'loaded 2 stories' in result.output
-    assert 'rendered graph as' in result.output
+
+    assert os.path.isfile('test.gv.svg')
+    with open('test.gv.svg') as output_file:
```

Comment:
Where is this file written? Would it make sense to apply the [`tmpdir` standard fixture](https://docs.pytest.org/en/latest/builtin.html#builtin-fixtures-function-arguments) here?